src/__tests__/Home.test.js

```diff
@@ -2,7 +2,21 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import Home from '../views/Home';
 
-it('renders welcome message', () => {
+it('renders the correnct contents', () => {
     render(<Home/>);
-    expect(screen.getByText('Welcome to Barefoot Nomad with styled component')).toBeInTheDocument();
-});
\ No newline at end of file
+    expect(screen.getByText('TRAVEL BOOKINGS')).toBeInTheDocument();
```

Comment:
@Patrick You run similar tests in App.test.js and Home.test.js, can you omit some codes?!

Reply:
Thanks @SamuelSabasNayo! Let me work on it.

Reply:
@SamuelSabasNayo! you can kindly review the work. I just kept the tests for my feature.